analytics/src/main/java/brocoeur/example/analytics/service/UserWinLossByGameService.java

```diff
@@ -51,27 +51,18 @@ private void updateAnalyticAccordingToWinOrLoss(final PlayerResponse playerRespo
             throw new IllegalStateException("UserWinLossByGame with gameId : " + gameId + " and userId : " + userId + " do not exists.");
         }
 
-        final List<Boolean> listOfIsWinner = playerResponse.getListOfIsWinner();
-
-        int updatedNumberOfWin = userWinLossByGame.getNumberOfWin();
-        int updatedNumberOfLoss = userWinLossByGame.getNumberOfLoss();
-        for (Boolean aBoolean : listOfIsWinner) {
-            if (Boolean.TRUE.equals(aBoolean)) {
-                updatedNumberOfWin += 1;
-            } else {
-                updatedNumberOfLoss += 1;
-            }
-        }
+        final int amountOfMoneyWon = playerResponse.getFinalAmount() - playerResponse.getInitialAmount();
```

Comment:
Name of variable doesn't fit well, maybe change to "amountChange" or something because in case of a loss there is no money being won

Reply:
This is the amount of money won to be added. Will review var names in next mr.